cli/cluster.go

```diff
@@ -113,20 +123,24 @@ func runCluster(args *docopt.Args) error {
 
 func runClusterAdd(args *docopt.Args) error {
 	s := &cfg.Cluster{
-		Name:   args.String["<cluster-name>"],
-		Key:    args.String["<key>"],
-		GitURL: args.String["--git-url"],
-		TLSPin: args.String["--tls-pin"],
+		Name:          args.String["<cluster-name>"],
+		Key:           args.String["<key>"],
+		GitURL:        args.String["--git-url"],
+		DockerPushURL: args.String["--docker-push-url"],
+		TLSPin:        args.String["--tls-pin"],
 	}
 	domain := args.String["<domain>"]
 	if strings.HasPrefix(domain, "https://") {
 		s.ControllerURL = domain
 	} else {
 		s.ControllerURL = "https://controller." + domain
 	}
-	if s.GitURL == "" {
+	if s.GitURL == "" && !args.Bool["--no-git"] {
 		s.GitURL = "https://git." + domain
 	}
+	if s.DockerPushURL == "" && args.Bool["--docker"] {
+		s.DockerPushURL = "https://docker." + domain
```

Comment:
If we only set this when the `--docker` flag is set, we should probably have another command that configures Docker so that you can set it up later? (and prompt if not configured and using a Docker command)